examples/rakutenunifiedads.amp.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
```

Comment:
Is this file unnecessary since ads.amp.html already contains your example?

Reply:
I'll delete this file